Resources/Prototypes/Maps/Pools/default.yml

```
@@ -1,7 +1,7 @@
 - type: gameMapPool
   id: DefaultMapPool
   maps:
- # - Arena # De-rotated, no current maintainer.
+  - Arena # Maintained by astriloqua.
```

YAMLlint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)